Doc/library/importlib.metadata.rst

```diff
@@ -301,6 +358,15 @@ function::
 Mapping import to distribution packages
 ---------------------------------------
 
+.. function:: packages_distributions()
+
+   Return a mapping from importable top level module and import package
+   names to the names of the distribution packages (if any) that provide
+   the corresponding files. To allow for namespace packages (which may
+   have members provided by multiple distribution packages), each module
+   name maps to a list of distribution names rather than mapping directly
+   to a single name.
```

Comment:
```suggestion
   Return a mapping from an importable top level module and its import package
   names to the associated names of the distribution packages (if any) that provide
   the corresponding files. To allow for namespace packages (which may
   have members provided by multiple distribution packages), each module
   name maps to a list of distribution names rather than mapping directly
   to a single name.
```

Reply:
Reworded in an attempt to make this clearer.